security.rst

```diff
@@ -2683,6 +2678,15 @@ like this:
   :doc:`impersonating </security/impersonating_user>` another user in this
   session, this attribute will match.
 
+.. note::
```

Comment:
Shall we remove this note entirely in 7.0 while unmerging?

Reply:
Yes, let's remove it 👍